src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/AdditionalParameterRegExTest.java

```diff
@@ -12,65 +17,65 @@
   public void testSingleParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:customer=Nickesocke");
-    assert (test1 != null);
-    assert ("customer".equals(test1.getKey()));
-    assert ("Nickesocke".equals(test1.getValue()));
+    assertNotNull(test1);
+    assertEquals("customer", test1.getKey());
+    assertEquals("Nickesocke", test1.getValue());
   }
 
   @Test
   public void testSpacedParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:customer=Nicke socke#");
-    assert (test1 != null);
-    assert ("customer".equals(test1.getKey()));
-    assert ("Nicke socke#".equals(test1.getValue()));
+    assertNotNull(test1);
+    assertEquals("customer", test1.getKey());
+    assertEquals("Nicke socke#", test1.getValue());
   }
 
   @Test
   public void testBlankParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:the_blank_parameteR=");
-    assert (test1 != null);
-    assert ("the_blank_parameteR".equals(test1.getKey()));
-    assert ("".equals(test1.getValue()));
+    assertNotNull(test1);
+    assertEquals("the_blank_parameteR", test1.getKey());
+    assertEquals("", test1.getValue());
```

Comment:
Isn't there some assertBlank? I'm sure there is a hamcrest matcher that does this

Reply:
`assertThat(test1.getValue(), isEmptyString());`